### ISSUE TITLE
Consent error value expected (consent_type)

### ISSUE THREAD

**Issue by dirceudn** [ReachFive/identity-android-sdk#46]
Following this code:

```
@Parcelize
data class Consent(
    val granted: Boolean,
    @SerializedName("consent_type")
    val consentType: Boolean?,
    val date: String
) : Parcelable
```
If you write the content like this:

`  val currentConsent = Consent(false, false, getCurrentDate())
`
`  consent["optin_newsletter"] = currentConsent
`
```
val profile = Profile(gender = gender, name = name, familyName = lastName,
                locale = address, birthdate = date, customFields = fields, consents = consent)

        reach5.updateProfile(token!!, profile = profile, success = {
            Timber.tag(TAG).d("Profile $it")
            handleSuccess()
        }, failure = { error ->
            handleError(error)

        })
```
We have the current error code:
`ReachFiveError(message=Bad Request, exception=null, data=ReachFiveApiError(error=invalid_request, errorDescription=Validation failed, errorDetails=[ReachFiveApiErrorDetail(field=consents.optin_newsletter.consent_type, message=String value expected)]))`

I think the correct value (consent_type)  in the object Consent must be a String not a boolean.

**Comment by roxanemace** [comment]
Thanks for the remark, I'll fix that right now.

**Comment by dirceudn** [comment]
👍

**Comment by roxanemace** [comment]
Hi @dirceudn, your bug was fixed in this release [v5.1.0](https://github.com/ReachFive/identity-android-sdk/blob/v5.1.0/CHANGELOG.md#v510-14082019). 😄

**Comment by dirceudn** [comment]
@roxanemace Thanks :D
